src/mococrw/key.h

```diff
@@ -92,6 +92,18 @@ class AsymmetricPublicKey
      * object as a result.
      */
     static AsymmetricPublicKey readPublicKeyFromHSM(HSM& hsm, const std::string& keyID);
+
+    static AsymmetricPublicKey genKeyOnHsmGetPublic(HSM& hsm,
```

Comment:
Kindly add doxygen comments for this function, and the other new ones.

Reply:
Done